src/main/java/hw36/AppConfig.java

```diff
@@ -0,0 +1,28 @@
+package hw36;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+@Configuration
+public class AppConfig {
+    @Autowired
+    private ProductRepository productRepository;
+
+    @Autowired
+    private Cart cart;
```

Comment:
You do not need to inject these beans here. Use them only when you need them. Remove please.